frontend_layout/data.json

```diff
@@ -0,0 +1 @@
+[{"uuidShedule":"10dff115d-61a1-43e4-8ab1-822d6db73260","graphName":"Graph 2", "state":"Started", "author":"молодцы","start":"-", "end":"-","tags":"ML"},{"uuidShedule":"9dff115d-61a1-43e4-8zb1-822d6db73260","graphName":"Graph 1", "state":"Planing", "author":"Alex","start":"-", "end":"-","tags":"NLP ML"},{"uuidShedule":"9dff115d-61a1-43e4-8ab1-822d6db73260","graphName":"Graph0","state":"planning","author":"0","start":"2018-11-17T11:25:03.189+0000","end":"2067-10-03T22:50:06.378+0000","tags":["0","1","2","3","4","5","6","7","8","9"]},{"uuidShedule":"5f9fca0f-a632-4741-8652-cf0af817794f","graphName":"Graph1","state":"planning","author":"1","start":"2018-11-17T11:25:03.189+0000","end":"2043-04-26T17:07:34.783+0000","tags":["0","1","2","3","4","5","6","7","8","9"]},{"uuidShedule":"b283ab7c-e39f-455f-9584-b69a2f775e80","graphName":"Graph2","state":"started","author":"2","start":"2018-11-17T11:25:03.189+0000","end":"2035-03-04T07:13:24.252+0000","tags":["0","1","2","3","4","5","6","7","8","9"]},{"uuidShedule":"d80bc49b-0369-4b16-a40f-4e32bfa3574f","graphName":"Graph3","state":"started","author":"3","start":"2018-11-17T11:25:03.189+0000","end":"2031-02-05T14:16:18.986+0000","tags":["0","1","2","3","4","5","6","7","8","9"]},{"uuidShedule":"1fae0fab-b94a-43c5-82da-f43b6d01ee60","graphName":"Graph4","state":"failed","author":"4","start":"2018-11-17T11:25:03.189+0000","end":"2028-08-26T23:18:03.826+0000","tags":["0","1","2","3","4","5","6","7","8","9"]},{"uuidShedule":"b8d84b5d-e442-4123-b81c-07a39e8a7b0a","graphName":"Graph5","state":"failed","author":"5","start":"2018-11-17T11:25:03.190+0000","end":"2027-01-09T21:19:13.721+0000","tags":["0","1","2","3","4","5","6","7","8","9"]},{"uuidShedule":"2e4c3b3c-ab20-41cf-b67f-d249041f318f","graphName":"Graph6","state":"succeed","author":"6","start":"2018-11-17T11:25:03.190+0000","end":"2025-11-10T19:54:20.788+0000","tags":["0","1","2","3","4","5","6","7","8","9"]},{"uuidShedule":"c278e63a-32e0-4e8b-bf6c-64f6bebb4f31","graphName":"Graph7","state":"succeed","author":"7","start":"2018-11-17T11:25:03.190+0000","end":"2024-12-27T00:50:41.088+0000","tags":["0","1","2","3","4","5","6","7","8","9"]},{"uuidShedule":"bd3c2098-1e37-4137-a9be-b3e270a06b6b","graphName":"Graph8","state":"canceled","author":"8","start":"2018-11-17T11:25:03.190+0000","end":"2024-04-23T02:01:10.211+0000","tags":["0","1","2","3","4","5","6","7","8","9"]},{"uuidShedule":"fbb7385a-c450-4f01-898b-add8ac78e499","graphName":"Graph9","state":"canceled","author":"9","start":"2018-11-17T11:25:03.190+0000","end":"2023-10-07T17:21:33.509+0000","tags":["0","1","2","3","4","5","6","7","8","9"]}]
```

Comment:
this is a test data
I think that this file should be placed into specific directory like "data/test/..."

Reply:
@avplatonov, you're right. I made a edits.